tests/Test/Krank/Checkers/IssueTrackerSpec.hs

```diff
@@ -75,8 +75,8 @@ giturlTests :: GitServer -> Spec
 giturlTests domain = do
   let domainName = serverDomain domain
   it "handles full https url" $ do
-    let match = check [fmt|https://{domainName}/guibou/krank/issues/1|]
-    match `shouldBe` Just (GitIssue domain "guibou" "krank" 1)
+    let match = check [fmt|https://{domainName}/guibou/krank/issues/2|]
```

Comment:
Why `2` ?

Reply:
why not, we have 1 everywhere. it kind of tests this is not hardcoded :D

Reply:
Good point. We may even write a quickcheck test to generate the test case automatically ;)